src/chord.rs

```diff
@@ -9,21 +11,34 @@ pub struct Barre {
     pub fret: u16,
 }
 
-pub const BARRE_FRET1: Barre = Barre {
-    from_string: 0,
-    to_string: 5,
-    fret: 1,
-};
+impl Barre {
+    pub const fn new(fret: u16) -> Self {
+        Self {
+            from_string: 0,
+            to_string: 5,
+            fret: fret,
+        }
+    }
+}
 
-pub const BARRE_FRET2: Barre = Barre {
-    from_string: 0,
-    to_string: 5,
-    fret: 2,
-};
+pub const BARRE_FRET1: Barre = Barre::new(1);
+pub const BARRE_FRET2: Barre = Barre::new(2);
+pub const BARRE_FRET3: Barre = Barre::new(3);
+
+pub const FRETBOARD: &str = "◯ ◯ ◯ ◯ ◯ ◯
+╒═╤═╤═╤═╤═╕
+│ │ │ │ │ │
+├─┼─┼─┼─┼─┤
+│ │ │ │ │ │
+├─┼─┼─┼─┼─┤
+│ │ │ │ │ │
+├─┼─┼─┼─┼─┤
+│ │ │ │ │ │
+└─┴─┴─┴─┴─┘";
 
 #[derive(Debug, Clone, Copy)]
 pub struct Chord<'a> {
-    pub short_name: &'a str,
+    pub short_names: [Option<&'a str>; 2],
```

Comment:
Let's change the type to the following?

```
Vec<String>
```

It is unlikely that we will have `[None, "foo"]` or `[None, None]`

Reply:
Same, I think we should change the type of `name` to `names` below with the same type for consistency.

Reply:
I gave it a try and I can make this work. TAL?

https://github.com/yzhong52/ascii_chord/pull/6